tests/features/project.feature

```diff
@@ -8,9 +8,10 @@ Feature: Project content creation
   Scenario: Creation of a Project content through the UI.
     Given I am logged in as a user with the "create oe_project content, access content, edit own oe_project content, view published skos concept entities, manage corporate content entities" permission
     And the following images:
-      | name    | file           | alt                |
-      | Image 1 | example_1.jpeg | Alternative text 1 |
-      | Image 2 | example_1.jpeg | Alternative text 2 |
+      | name    | file            | alt                 |
+      | Image 1 | example_1.jpeg  | Alternative text 1  |
+      | Image 2 | example_2.png   | Alternative text 2  |
+      | Image 3 | example_3.png   | Alternative text 3  |
```

Comment:
can't we reuse 'example_1.jpeg'? we have 'featured media form element' ..

Reply:
Fixed